doc/dynamic-environments/git-environments.mkd

```diff
@@ -45,6 +45,16 @@ future r10k runs. Finally, if there are directories that do not match existing
 branches, r10k will assume that the branches for those environments were delete
 and will remove those environments.
 
+r10k will need to be be able to authenticate with each repository. Most Git
+systems support authentication with SSH keys. GitHub calls them [deploy
+keys][github-deploy-keys]. Bitbucket calls them [deployment
+keys][bitbucket-deployment-keys]. Stash calls them [SSH access
```

Comment:
Looking at it again, it looks like the documentation for Stash is weird and they're actually called access keys.